test/app-tap/datetime.test.lua

```diff
@@ -1830,6 +1830,565 @@ test:test("Parse strptime format", function(test)
     end
 end)
 
+-- Parse string with a custom format, then format datetime object
+-- to a string with the same format and compare initila and
+-- resulted strings:
+--
+--   dt = datetime.parse(<str>, { format = <fmt> })
+--   dt:format(<fmt>) == <str>
+--
+test:test("Parse strptime format and check with :format()", function(test)
+    local formats = {
+        -- %a
+        -- Works for 'Thu' only when date is not complete.
```

Comment:
Don't get the comment. Do you mean specific day of the weak or 3-chars acronym, and this is an example?

Reply:
For a string with omitted year/month/day we complement these fields by appropriate values from a Unix time.
Weekday in a Unix time was Thursday. So when all aforementioned fields are missed parse will return `Thu`:

```lua
tarantool> datetime.parse('Sat', {format = '%a'}):format('%a')
---
- Thu
...

tarantool> datetime.parse('1970 01 03 Sat', {format = '%Y %m %d %a'}):format('%Y %m %d %a')
---
- 1970 01 03 Sat
...

tarantool> 
```

I've marked such cases with conversion specifications as "date is not complete".

Reply:
Is there a ticket for that or should we create one?

Reply:
It will be nice to mention #10470 here, then.

Reply:
Added.